libraries/support/azure/vault.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Azure
+  class Vault
+    include Service
+
+    def initialize
+      @required_attrs = %i(rest_client)
+      @page_link_name = 'nextLink'
+    end
+
+    def keys
+      get(
+        url: '/keys',
+        api_version: '2016-10-01',
+        use_cache: false,
+      )
+    end
+
+    def key(key_name, key_version)
+      get(
+        url: "/keys/#{key_name}/#{key_version}",
+        api_version: '2016-10-01',
+        use_cache: false,
+      )
+    end
+
+    def key_versions(key_name)
+      get(
+        url: "/keys/#{key_name}/versions",
+        api_version: '2016-10-01',
+        use_cache: false,
+      )
+    end
+
+    def secrets
+      get(
+        url: '/secrets',
+        api_version: '2016-10-01',
+        use_cache: false,
+      )
+    end
+
+    def secret(secret_name, secret_version)
+      get(
+        url: "/secrets/#{secret_name}/#{secret_version}",
+        api_version: '2016-10-01',
+        unwrap: unwrap,
```

Comment:
Why do we need `unwrap` here? For graph we are using unwrap since things are paginated.

Reply:
The default unwrap (https://github.com/inspec/inspec-azure/blob/Add-Keys/libraries/support/azure/service.rb#L86) looks for an attribute `value` in the response, which with Azure usually contains the JSON representation of what we want to map to struct. 
Unfortunately for Keys, there is an actual attribute called `value`, and so instead of mapping the entire response, it mapped only this `value` field to struct.

Reply:
Thanks!